keras/ops/numpy_test.py

```diff
@@ -4733,3 +4733,26 @@ def test_eye(self, dtype):
             ),
             standardize_dtype(jnp.eye(3, 4, 1, dtype=dtype).dtype),
         )
+
+
+class NumpyOutputDtypesTest(testing.TestCase, parameterized.TestCase):
```

Comment:
Hi @SuryanarayanaY 

Just came across this PR :)

Instead of creating new class, I believe you can add these tests to `NumpyDtypeTest` which is designed to verify both input and output dtypes

https://github.com/keras-team/keras/blob/21a3c0b7ce2bf738d81e3737aedba1a1ebd93c8b/keras/ops/numpy_test.py#L4306

Reply:
Hi @james77777778 , Yeah it can be done there. But the Doc string states this class is to verify Jax behaviour.Hence I thought of creating a new class.